src/App.jsx

```diff
@@ -5,11 +5,9 @@ import Routes from './Routes.jsx';
 class App extends Component {
     render () {
         return (
-            <div className='container-fluid'>
-                <div className="row">
+            <div >
```

Comment:
If a `<div>` has no style associated with it, but is only there because JSX elements must be wrapped in an enclosing tag, consider using [fragments](https://reactjs.org/docs/fragments.html).